### PR TITLE
Moving away from Sapper

### PR DESCRIPTION
**Pull request by martonlederer**
Sapper is now officially dead or on it's way to die xD. Routify is a great routing wrapper for Svelte, that includes Static Site Rendering, and can be a perfect solution for us to introduce something with faster builds/development, while also solving our problem of deploying to arweave.
This pr includes:
- [x] Removing Sapper
- [x] Switching to Routify
- [x] Using `nollup` for ultra-fast development

TODO: fix errors

### CODE REVIEW

File: src/stores/themeStore.ts [hunk 1]
```diff
@@ -29,8 +28,6 @@ export const displayTheme = derived(theme, ($theme) => {
       break;
     case Theme.Auto:
       if (
-        // @ts-ignore
-        process.browser &&
```
Comment (littledivy) [inline src/stores/themeStore.ts:33]: YES! FINALLY!

File: package.json [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
   "version": "0.0.1",
   "scripts": {
-    "dev": "sapper dev",
-    "build": "sapper build --legacy",
-    "export": "sapper export --legacy --entry \"/ trade tokens post login gallery app 404 app/all-exchanges app/all-transactions\"",
-    "start": "node __sapper__/build",
+    "dev": "run-p routify nollup",
+    "build": "run-s build:*",
+    "build:app": "routify -b && rollup -c",
+    "build:static": "spank",
```
Comment (maximousblk) [inline package.json:8]: to make the CI and vercel happy:
```suggestion
    "export": "spank",
```

**Review by maximousblk — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (martonlederer) [reply package.json:8]: The vercel config will have to be updated anyways tho

Comment (maximousblk) [reply package.json:8]: hmm... yeah

**Comment by maximousblk** [comment]
Close this?

**Pull request closed without merging**